src/quests.json

```diff
@@ -37,10 +37,10 @@
                     -75.18278002738953
                 ], 
                 "historicContext": {
-                    "text": "By the 1840s the chaste, neoclassical architecture of the buildings, the landscaped garden, and the promenades up to and around the reservoirs on \"Fair Mount,\" as the rise was identified as early as 1687 on a map of Philadelphia, were major attractions. Ornamental sculptures, fountains, and airy gazebos added to the visual pleasure of the site.", 
+                    "text": "By the 1840s the chaste, neoclassical architecture of the buildings, the landscaped garden, and the promenades up to and around the reservoirs on \"Fair Mount,\"&#8230; were major attractions. Ornamental sculptures, fountains, and airy gazebos added to the visual pleasure of the site.", 
                     "attribution": "Jane Mork Gibson"
                 }, 
-                "radius": 5, 
+                "radius": 4, 
```

Comment:
Consider updating the commit message to note that the radius values are changing here because this file wasn't regenerated as part of https://github.com/azavea/pwd-waterworks-revealed/pull/159.